src/App/components/FactsComponentAdder/FactsComponentAdder.js

```diff
@@ -1,20 +1,20 @@
-import React, { useState } from 'react';
+import React, { useState } from 'react'
 import './FactsComponentAdder.css'
 import { CounterAdder } from '../CounterAdder/CounterAdder'
 import { FactsLoader } from '../FactsLoader/FactsLoader'
 import { PusherToFirebase } from '../FirebaseComponents/PusherToFirebase/PusherToFirebase'
 
-export function FactsComponentAdder({ userId }) {
-  const [factValue, setFactValue] = useState(1);
-  const [facts, setFacts] = useState('');
+export function FactsComponentAdder ({ userId }) {
+  const [factValue, setFactValue] = useState(1)
```

Comment:
shouldn't this be `0`?
```suggestion
  const [factValue, setFactValue] = useState(0)
```